.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+**Checklist**
+- [x] Bug Report - The issue you are reporting might be there already. Please search on the [issues tracker](https://github.com/openfoodfacts/openfoodfacts-server/issues) before creating or reporting a issue.
```

Comment:
I'm not sure what's correct here. Are the checkboxes to be pre-checked for the user? They are not for the PR template.

Reply:
Yes they need not to be prechecked the user has to precheck it doing so :+1: